src/rooms/entities/room-type.entity.ts

```diff
@@ -0,0 +1,21 @@
+import Project from 'src/projects/entities/project.entity';
+import {
+    Column,
+    Entity,
+    JoinColumn,
+    ManyToOne,
+    PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity('room_types')
+export default class RoomType {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column({ length: 50 })
+    name: string;
+
+    @ManyToOne(() => Project, (project) => project.room_types)
```

Comment:
If you use refactoring, vscode will take care of this for you.

```suggestion
    @ManyToOne(() => Project, (project) => project.roomTypes)
```